index.html

```diff
@@ -35,6 +36,99 @@ <h2>
+            <div class="boxes white">
+                <div>css</div>
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+            <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+    </div>
+    <div class="aboutme">
+        <h2>About me</h2>
+        <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <div>Get my resume</div>
```

Comment:
The button (e.g., "Get my resume") should be a link `a` or `button` tag. Kindly fix it to meet the expected requirements 📝✔